services/search/api.py

```diff
@@ -99,12 +100,33 @@ def to_representation(self, obj):
                 representation["department"] = DepartmentSerializer(obj.department).data
                 if obj.geometry:
                     representation["geometry"] = munigeo_api.geom_to_json(
-                            obj.geometry, DEFAULT_SRS
-                        )
+                        obj.geometry, DEFAULT_SRS
+                    )
                 else:
                     representation["geometry"] = None
-                    
-            if object_type == "address" or object_type == "unit":
+
+            if object_type == "address":
+                representation["number"] = getattr(obj, "number", "")
+                representation["number_end"] = getattr(obj, "number_end", "")
+                representation["letter"] = getattr(obj, "letter", "")
+                representation["modified_at"] = getattr(obj, "modified_at", "")
+                municipality = {}
+                municipality["id"] = getattr(obj.street, "municipality_id", "")
+                municipality["name"] = {}
+                municipality["name"]["fi"] = getattr(
+                    obj.street.municipality, "name_fi", ""
+                )
+                municipality["name"]["sv"] = getattr(
+                    obj.street.municipality, "name_sv", ""
+                )
+                representation["municipality"] = municipality
+                street = {}
```

Comment:
This could be replaced with dict object literal:
````

street = {"name": {}}
```

Reply:
fixed.